plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1868,6 +1868,10 @@ public void testConstantUpdateWithVarcharEqualityPredicates()
             }
             assertUpdate("UPDATE " + table.getName() + " SET col1 = 20 WHERE col2 = 'A'", 1);
             assertQuery("SELECT * FROM " + table.getName(), "VALUES (1, 'a'), (20, 'A')");
+
+            // test update set to null
+            assertUpdate("UPDATE " + table.getName() + " SET col1 = null WHERE col2 = 'A'", 1);
```

Comment:
A predicate isn't necessary for https://github.com/trinodb/trino/issues/24204 and we should test with all connectors. Please add a new test to `BaseConnectorTest` instead.